Description: The example certificate uses DSAWithSha1 as signature algorithm, but according
to the documentation, that one is not supported.
diff --git a/node-authorizer/pkg/authorizers/aws/authorizer_test.go b/node-authorizer/pkg/authorizers/aws/authorizer_test.go
index df16d57..cf1ce61 100644
--- a/node-authorizer/pkg/authorizers/aws/authorizer_test.go
+++ b/node-authorizer/pkg/authorizers/aws/authorizer_test.go
@@ -43,6 +43,7 @@ func newTestAuthorizer(t *testing.T, config *server.Config) *awsNodeAuthorizer {
 }
 
 func TestValidateIdentityDocument(t *testing.T) {
+	t.Skip("DSAWithSHA1 is not supported by crypto.x509")
 	c := newTestAuthorizer(t, nil)
 
 	request := &Request{
